moped-database/metadata/actions.graphql

```diff
@@ -0,0 +1,2 @@
+
+
```

Comment:
What's going on with this file? Seems weird that two blank lines were added.

Reply:
I have observed this happen more than a handful of times. With some rhyme or reason all to itself, the `graphql-engine` process will write out these two lines when the console is used to create migrations / change the schema. I don't think they do any harm; i think of them as similar to leaving a console.log() in a JS app. I'd be genuinely surprised if this hasn't been fixed up in a more recent version of the graphql endpoint daemon.

Reply:
Yes, as a general rule, I would only commit files or chunks of code that I've actively changed and intend to be in the PR.